docs/preview/features/powershell/keyvault.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Scripts related to interacting with Azure Key Vault"
+layout: default
+---
+
+# Azure Key Vault
+
+## Installation
+
+To have access to the following features, you have to import the module:
+
+```powershell
+PS> Import-Module -Name Arcus.Scripting.PS.KeyVault
```

Comment:
Let's remove all of these `.PS` prefixes as discussed
```suggestion
PS> Import-Module -Name Arcus.Scripting.KeyVault
```

Reply:
Removed all `PS` path's in project.